packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -2,18 +2,21 @@ import { Patient } from "@metriport/core/domain/patient";
 import { Coordinates } from "@metriport/core/external/aws/location";
 import convert from "convert-units";
 import { Sequelize } from "sequelize";
+import { Config } from "../../../../shared/config";
 import { CQDirectoryEntryModel } from "../../models/cq-directory";
 
 export const DEFAULT_RADIUS_IN_MILES = 50;
+const cqIgnoreList: string[] = constructGatewayIgnoreList();
```

Comment:
hyper nit: I would call this exclude not ignore. Ignore would be like requests coming in and we ignore them , but doesnt make as much sense to say ignore when talking about us choosing orgs to send requests to